utils/utils.tsx

```diff
@@ -95,14 +106,14 @@ export function createSafeId(prop) {
     return str.replace(new RegExp(' ', 'g'), '_');
 }
 
-export function cmdOrCtrlPressed(e, allowAlt = false) {
+export function cmdOrCtrlPressed(e: React.KeyboardEvent | KeyboardEvent, allowAlt = false) {
```

Comment:
Nothing to change here, but I wish we knew which of these types was correct to use. We've ended up mixing them in a lot of places